app/stylesheet/automate-simulation.scss

```diff
@@ -0,0 +1,46 @@
+.automate-simulation-page {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    .automate-simulation-form-wrapper {
+        border-right: 0.5px solid black;
+        padding-right: 2%;
+    }
+
+    .automate-simulation-summary-wrapper {
+        width: 67%;
+        padding-left: 2%;
+        padding-top: 2%;
+
+        .simulation-title-text {
+            font-size: x-large;
+            font-weight: lighter;
+        }
+    }
+
+    @for $i from 1 through 5 {
+        #attributeValuePairLabel_#{$i} {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            margin-top: 1%;
+        }
+    }
+    @for $i from 1 through 5 {
+        #subform_#{$i} {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            gap: 5%;
+        }
```

Comment:
we can avoid this loop by using a simple classname, probably like `.attributeValuePairLabel` and `.subform`

Reply:
this would generate 10 css definitions. the best approach would be to add a classname :)